data_handler/handlers/loan_states/zklend/run.py

```diff
@@ -107,14 +112,16 @@ def run(self) -> None:
         """
         max_retries = 5
         retry = 0
-        zklend_protocol_address = self.PROTOCOL_ADDRESSES[0]
+        zklend_protocol_address = self.PROTOCOL_ADDRESSES
```

Comment:
It seems to be a set. Would not we get in trouble using it as input for `get_data` method?

Reply:
<img width="742" alt="Screenshot 2024-08-02 at 8 53 05" src="https://github.com/user-attachments/assets/e2d56aba-21ea-4a02-94f3-082f0e1f9c6d">
for some protocols it works as set, for some of them as string.
Answer on your question, in this case -> No.

p.s: I reworked this logic in the past, because there are only 1 address per protocol what makes more sense to use string instead of set